vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -0,0 +1,210 @@
+            :src="blog.thumbnail"
+            :class="
+              blogs.length == 2
+                ? 'tw-top-[11.76rem] 2xl:tw-top-[8.625rem]'
+                : 'tw-top-[14.2rem] lg:tw-top-[13.3rem] 2xl:tw-top-[10.625rem]'
+            "
+            class="tw-hidden md:tw-block tw-absolute tw-right-10 tw-w-[20rem] 2xl:tw-w-[31.875rem] tw-max-h-[13.75rem] 2xl:tw-max-h-[20rem] tw-animate-fadeIn"
+            loading="lazy"
+            :alt="blog.title"
+          />
+
+          <div class="tw-flex tw-flex-row tw-gap-8">
+            <div class="tw-flex tw-flex-col tw-pt-[1.5rem] md:tw-pt-[2.120rem]">
+              <div
+                class="tw-text-black-core/[.87] tw-font-inter-semibold md:tw-font-inter-regular tw-text-[1.5rem] md:tw-text-[2.188rem] 2xl:tw-text-[3.125rem] tw-leading-[1.95rem] md:tw-leading-[2.183rem] 2xl:tw-leading-[4.063rem]"
+              >
+                {{ blog.pubDate[1] }}
+              </div>
+              <div
+                class="tw-text-center tw-text-black-core/[.87] font-inter-medium md:tw-font-inter-regular tw-text-[0.375rem] md:tw-text-[0.563rem] 2xl:tw-text-[0.75rem] tw-leading-[0.4875rem] md:tw-leading-[0.688rem] 2xl:tw-leading-[0.975rem]"
+              >
+                {{ blog.pubDate[0] }}
+              </div>
+            </div>
+            <div class="tw-flex tw-flex-col">
+              <div
+                class="tw-mt-[0.9375rem] md:tw-mt-[1.813rem] tw-w-[16.5rem] md:tw-w-[19.5rem] lg:tw-w-[27.5rem] 2xl:tw-w-[43.875rem] tw-text-black-core/[.87] tw-font-inter-semibold tw-text-[1.125rem] md:tw-text-[1.688rem] 2xl:tw-text-[2.125rem] tw-leading-[1.575rem] md:tw-leading-[2.313rem] 2xl:tw-leading-[3.188rem]"
+              >
+                <span
+                  class="tw-pb-[0.3rem] md:hover:tw-bg-gradient-to-b hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-bg-[length:100%_5px] hover:tw-bg-no-repeat hover:tw-bg-bottom"
+                >
+                  {{ blog.title }}</span
+                >
+              </div>
+              <div
+                class="tw-flex tw-flex-row tw-gap-[9rem] lg:tw-gap-72 2xl:tw-gap-[31rem]"
+              >
+                <div
+                  class="tw-mt-[0.9375rem] tw-w-[4.5rem] 2xl:tw-w-[5rem] tw-text-black-core/[.87] tw-font-inter-medium tw-text-[0.625rem] md:tw-text-[0.875rem] 2xl:tw-text-[1.125rem] tw-leading-[0.8125rem] md:tw-leading-[1.125rem] 2xl:tw-leading-[1.463rem]"
+                >
+                  {{ blog.author }}
+                </div>
+                <div
+                  class="tw-mt-[0.9375rem] tw-w-[3.25rem] md:tw-w-[4.5rem] 2xl:tw-w-[5.938rem] tw-text-black-core/[.87] tw-font-inter-medium tw-text-[0.625rem] md:tw-text-[0.875rem] 2xl:tw-text-[1.125rem] tw-leading-[0.8125rem] md:tw-leading-[1.125rem] 2xl:tw-leading-[1.463rem]"
+                >
+                  Read more
+                </div>
+              </div>
+            </div>
+          </div>
+          <hr
+            v-if="
```

Comment:
I think this condition can be reducible. If I see at figma, Each blog should contain either the top or bottom line and add one extra line to the top or bottom of all blogs.

Reply:
this lines are different in case of mobile and desktop that's why have to use 2 conditions